### PR TITLE
[1LP][RFR] add test_appliance_contains_ansible_modules

### PR DESCRIPTION
**Pull request by jaryn**
Adding test checking presence of ansible modules

{{pytest: cfme/tests/test_appliance.py::test_appliance_contains_ansible_modules -v}}

### CODE REVIEW

File: cfme/tests/test_appliance.py [hunk 1]
```diff
@@ -373,6 +373,25 @@ def test_appliance_exec_scripts():
...
+    check that there are ansible modules included in the appliance.
+
+    Polarion:
+        assignee: jhenner
+        casecomponent: Appliance
+        caseimportance: high
+        initialEstimate: 1/6h
+        startsin: 5.9.2
+    Bugzilla:
+        1678130
```
Comment (digitronik) [inline cfme/tests/test_appliance.py]: This test is for automating RFR right? Please add `automates` meta if possible.

Comment (jaryn) [reply cfme/tests/test_appliance.py]: Oh, I forgot. THanks

File: cfme/tests/test_appliance.py [hunk 2]
```diff
@@ -373,6 +373,26 @@ def test_appliance_exec_scripts():
...
+    """
+    assert appliance.ssh_client.run_command(
+        'stat /opt/rh/cfme-gemset/bundler/gems/cfme-conten*/content/ansible/roles'
```
Comment (sbulage) [inline cfme/tests/test_appliance.py]: Along with this, You can also check for gems is available or not like:
```
[root@localhost vmdb]# bundle exec gem list | grep ansible
ansible_tower_client (0.20.1)
manageiq-providers-ansible_tower (0.1.0)
```

Comment (jaryn) [reply cfme/tests/test_appliance.py]: @sbulage I am not sure it is a good idea. If I understand the bug, the package containing the files may be different depending on are we testing MiQ or CFME.

Also, the title of the BZ is to make sure the modules are there. It is not about checking some package is present. The package may contain the modules, or may not. Testing for the files presence is more specific.

Comment (sbulage) [reply cfme/tests/test_appliance.py]: Okay.

**Pull request merged**